src/db_iter.h

```diff
@@ -96,17 +109,23 @@ class TitanDBIterator : public Iterator {
       std::unique_ptr<BlobFilePrefetcher> prefetcher;
       status_ = storage_->NewPrefetcher(index.file_number, &prefetcher);
       if (status_.IsCorruption()) {
+        // If use `DeleteFilesInRanges`, we may encounter this failure,	
+        // because `DeleteFilesInRanges` may expose an old key which	
+        // corresponding blob file has already been GCed out, so we	
+        // cannot abort here.
         fprintf(stderr,
                 "key:%s GetBlobValue err:%s with sequence number:%" PRIu64 "\n",
                 iter_->key().ToString(true).c_str(), status_.ToString().c_str(),
                 options_.snapshot->GetSequenceNumber());
+        return false;
```

Comment:
change the `fprintf` to print to info log? And I think we shall print only once for each iterator.